### DISCUSSION TITLE
How to update Cookie expiry to 30 days from 20 years

### DISCUSSION THREAD

**Discussion by leo7721** [ServiceStack/Discuss#67] [Q&A]
Currently, SS-PID Cookie is of 20 years expiry by default. Is it possible to edit the expiry time to 30 days?

Thanks in advance.

**Comment by mythz** [comment]
It's a "permanent" cookie, it's supposed to never expire, which other features should be able to depend on to attach permanent data they want against the browser.

It wasn't overridable before but I've just made it configurable in [this commit](https://github.com/ServiceStack/ServiceStack/commit/e38dbe0ae269ccf1bf849fcbcd3efcdb6f6d4ca3) with:

```csharp
ServiceStack.Host.Cookies.PermanentCookieExpiry = DateTime.UtcNow.AddDays(30);
```

Note: this changes the expiration of all permanent cookies.

This change is available from v6.9.1 that's now [available on MyGet](https://docs.servicestack.net/myget).
